taste_tester.gemspec

```diff
@@ -1,7 +1,9 @@
+# frozen_string_literal: true
+
 # rubocop:disable Metrics/BlockLength
 Gem::Specification.new do |s|
   s.name = 'taste_tester'
-  s.version = '0.0.13'
+  s.version = '0.0.14'
```

Comment:
please do not do version bumps with other changes.

Reply:
I'll back this bump out.